integration/integration_cli_test.go

```diff
@@ -102,7 +102,7 @@ cd /test
 	assert.Equal(t, 0, code)
 }
 
-// Dir copies a directory
+// SetDir copies a directory
```

Comment:
The name in the docu here and the func name are out of sync. Was out of sync actually also before ...